.gitignore

```diff
@@ -1,3 +1,7 @@
-dist
+dist/*
 .vscode
-.aider*
+.clang-format
```

Comment:
I don't think we want to ignore this: it is already committed, having been created by ufbt.